File: 2° Semestre/Lista Java/orcamento.java
import java.util.Scanner;
class Main 
{
  
  public static void main(String[] args) 
  {
    Scanner ler= new Scanner(System.in);
    float sal, gasto;
    System.out.println("Qual seu salário:");
    sal= ler.nextFloat();
    System.out.println("Qual o valor gasto no mês:");
    gasto = ler.nextFloat();

    if (gasto<=sal)System.out.println("Gastos no Orçamento.");
    else System.out.println("Orçamento estoura.");
  
  }
}

File: 2° Semestre/Lista C/orcamento.c
#include <stdio.h>

int main(void) 
{
  float sal, gasto;
  printf("Qual seu salário:\n");
  scanf("%f",&sal);
  printf("Qual o valor gasto no mês:\n");
  scanf("%f",&gasto);

  if (gasto<=sal)printf("Gastos no Orçamento.");
  else printf("Orçamento estoura.");
  
  return 0;
}